src/screens/ProfileScreen/index.js

```diff
@@ -166,7 +166,7 @@ const Header = (props) => {
             <Text style={styles.postText(profileTabIndex === TAB_INDEX_SIGNED)}>Signed Posts</Text>
           </CustomPressable>
           <CustomPressable
-            style={styles.tabItem(profileTabIndex === TAB_INDEX_ANONYMOUS)}
+            style={styles.tabItem(profileTabIndex === TAB_INDEX_ANONYMOUS, true)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `true` parameter in the `styles.tabItem` function call is a good change as it allows for conditional styling based on whether the tab is anonymous or not. However, it would be more readable and maintainable if you used a constant instead of a boolean literal. This would make the code self-documenting and easier to understand.
```diff
- style={styles.tabItem(profileTabIndex === TAB_INDEX_ANONYMOUS, true)}
+ const IS_ANONYMOUS = true;
+ style={styles.tabItem(profileTabIndex === TAB_INDEX_ANONYMOUS, IS_ANONYMOUS)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->